### PR TITLE
modify range_q method

### PR DESCRIPTION
**Pull request by AurelienRenou**
modified the range_q method, it used to use a method to look up the dof id in the model and get the ranges that match, now we just need to have the biorbd model and the method just sweep all the segments and the dof of it and gets the ranges.

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -4,69 +4,27 @@
...
-        range_max = biorbd_model.segment(seg_id).QRanges()[dof_id].max()
-        min_ranges[dof_nb] = range_min
-        max_ranges[dof_nb] = range_max
-    return min_ranges, max_ranges
-
-
-def get_segment_and_dof_id_from_global_dof(biorbd_model: biorbd.Model, global_dof: int):
-    """
-    Allow the users to get the segment id which correspond to a dof of the model and the id of this dof in the segment
+    Give the ranges of the model
```
Comment (Ipuch) [inline utils.py]: The ranges of generalized coordinates q

File: utils.py [hunk 2]
```diff
@@ -4,69 +4,27 @@
...
-    dof = dof.replace("_", "")  # we remove the _ "_DofName"
-    count_dof = 0
-    while biorbd_model.segment(seg_id).nameDof(count_dof).to_string() != dof:
-        count_dof += 1
-
-    return seg_id, count_dof
+    q_range_max = []
+    q_range_min = []
+    for i in range(biorbd_model.nbSegment()):
+        segment = biorbd_model.segment(i)
```
Comment (Ipuch) [inline utils.py]: is it possible to do :

```python3
for seg in biorbd_model.segments():
        q_range_max += [q_range.max() for q_range in seg.QRanges()]

**Review by Ipuch — COMMENTED** [review]
two tiny details ;)

File: utils.py (hunk 1 above)
Comment (AurelienRenou) [reply utils.py]: done

File: utils.py (hunk 2 above)
Comment (AurelienRenou) [reply utils.py]: done

**Pull request merged**